docs/source/deploying.rst

```diff
@@ -84,59 +84,113 @@ The following resources explain how to set up Dask on a variety of local and dis
 
 Cloud
 -----
-|Coiled|_ **is recommended for deploying Dask on the cloud.**
-Though there are other options you may consider depending on your specific needs:
 
-- `Coiled <https://coiled.io?utm_source=dask-docs&utm_medium=deploying>`_: Commercial Dask deployment option, which handles the creation and management of Dask clusters on cloud computing environments (AWS, GCP, and Azure).
-- `Dask Cloud Provider <https://cloudprovider.dask.org/en/latest/>`_: Constructing and managing ephemeral Dask clusters on AWS, DigitalOcean, Google Cloud, Azure, and Hetzner.
-- `Dask-Yarn <https://yarn.dask.org>`_: Deploy Dask on YARN clusters, such as are found in traditional Hadoop installations.
+Deploying on commercial cloud like AWS, GCP, or Azure is convenient because you can quickly scale out to many machines for just a few minutes, but also challenging because you need to navigate awkward cloud APIs, manage remote software environments with Docker, send data access credentials, make sure that costly resources are cleaned up, etc..  The following solutions help with this process.
+
+-   `Coiled (recommended) <https://coiled.io?utm_source=dask-docs&utm_medium=deploying>`_:
+    this commercial SaaS product handles most of the deployment pain we currently
+    see, is easy to use, and quite robust.  The free tier is generous enough
+    for most individual users, even for those who don't want to engage with a
+    commercial company.  The API looks like the following.
+
+    .. code-block:: python
+
+       import coiled
+       cluster = coiled.Cluster(
+           n_workers=100,
+           region="us-east-2",
+           worker_memory="16 GiB",
+           spot_policy="spot_with_fallback",
+       )
+       client = cluster.get_client()
+
+- `Dask Cloud Provider <https://cloudprovider.dask.org/en/latest/>`_: a pure and simple OSS solution that sets up Dask workers on cloud VMs, supporting AWS, GCP, Azure, and also other commercial clouds like Hetzner and Digital Ocean.
+
+- `Dask-Yarn <https://yarn.dask.org>`_: deploys Dask on legacy YARN clusters, such as can be set up with AWS EMR or Google Cloud Dataproc
```

Comment:
[Non-blocking comment] You have a much better sense for if the term "legacy" applies here. I definitely see less Dask traffic around YARN clusters these days -- just noting that this is the first time I'm seeing the term "legacy" in this context.

Reply:
Yeah, I'm comfortable throwing some shade at Yarn today.